src/datatypes/Card.ts

```diff
@@ -75,6 +75,7 @@ export interface CardDetails {
   cubeCount?: number;
   pickCount?: number;
   isExtra?: boolean;
+  wordCount?: number;
```

Comment:
With the addition to the type it would be good to add to the logic within update_cards.ts convertCard() function. Otherwise the field will be undefined unless the card has passed through the word count sort.